src/components/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+
+const TodoItem = (props) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  const completedStyle = {
+    fontStyle: 'italic',
+    color: '#595959',
+    opacity: 0.4,
+    textDecoration: 'line-through',
+  };
+
+  const { completed, id, title } = props.todo;
+
```

Comment:
With reference to the above request, you can fix it here by adding prop-types like this
```suggestion
  const { completed, id, title } = props.todo;
TodoItem.propTypes = {
 completed: PropTypes.bool.isRequired,
 id: PropTypes.number.isRequired,
 title: PropTypes.stringl.isRequired,
};
```